vernac/record.ml

```diff
@@ -596,7 +590,7 @@ let declare_structure ~cumulative finite ~ubind ~univs ~variances paramimpls par
   in
   let impls = List.map (fun _ -> paramimpls, []) record_data in
   let kn = DeclareInd.declare_mutual_inductive_with_eliminations mie ubind impls
-      ~primitive_expected:(primitive_flag ())
+      ~primitive_expected:primitive_proj
```

Comment:
Should I rename this label to `primitive_proj` to be more homogeneous?

Reply:
I'd say so, but maybe in another PR